src/Clockify/ClockifyMessageSource.cs

```diff
@@ -27,18 +31,27 @@ public ClockifyMessageSource(IStringLocalizer<ClockifyMessageSource> localizer)
         public string TaskSelectionQuestion => GetString(nameof(TaskSelectionQuestion));
         public string NewTask => GetString(nameof(NewTask));
         public string No => GetString(nameof(No));
-        
+
         public string ReportDateRangeError => GetString(nameof(ReportDateRangeError));
         public string ReportWorkedPeriodUnrecognized => GetString(nameof(ReportWorkedPeriodUnrecognized));
         public string ReportTotalHours => GetString(nameof(ReportTotalHours));
         public string ReportNoWork => GetString(nameof(ReportNoWork));
         public string ReportWork => GetString(nameof(ReportWork));
         public string ReportDateRangeExceedOneYear => GetString(nameof(ReportDateRangeExceedOneYear));
-        
+
         public string RemindStoppedAlready => GetString(nameof(RemindStoppedAlready));
         public string RemindStopAnswer => GetString(nameof(RemindStopAnswer));
+
         public string RemindEntryFill => GetString(nameof(RemindEntryFill));
 
-        private string GetString(string name) => _localizer[name];
+        private string GetString(string name)
+        {
+            string? key = _localizer.GetAllStrings()
+                .Select(s => s.Name)
+                .Where(k => k.StartsWith(name))
+                .OrderBy(_ => Rnd.Next())
```

Comment:
How does it works? It's not clear

Reply:
I take all possible resource values that starts with the target key, and pick a random one

Reply:
I am referring only to the order by clause

Reply:
you order by an expression whose values are random => you shuffle